package.json

```diff
@@ -27,6 +27,7 @@
     "handlebars": "^4.0.10",
```

Comment:
we can simplify the dependencies here. We are not using Hapi anymore so we could have just "live-server" as a dependency